### DISCUSSION TITLE
Code review IM-13

### DISCUSSION THREAD

**Discussion by yarman1** [HowProgrammingWorks/LiveQA#110] [Code review]
Yaroslav Hordiienko IM-13
I'm working on a simple IP address parser for Node js. The validation system and parser are mostly ready, as well as the tests for them. Check ip-manipulator.js file and test folder please.
[My repository](https://github.com/yarman1/ip-manipulator)

**Comment by tshemsedinov** [comment]
https://github.com/yarman1/ip-manipulator/issues/11
